dkarlovi/symfony-flex-docker/0.1/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "copy-from-package": {
+        ".infra/": ".infra",
```

Comment:
Nope, it's in [the package](https://github.com/dkarlovi/symfony-flex-docker) from which this recipe is copying.